ballerina/http_client_connection_pool.bal

```diff
@@ -27,11 +27,13 @@ configurable int maxActiveStreamsPerConnection = 100;
 # + maxIdleConnections - Maximum number of idle connections allowed per pool.
 # + waitTime - Maximum amount of time (in seconds), the client should wait for an idle connection before it sends an error when the pool is exhausted
 # + maxActiveStreamsPerConnection - Maximum active streams per connection. This only applies to HTTP/2. Default value is 100
+# + evictableIdleTime - Evictable time for an idle connection in seconds. Default value is 5 minutes
 public type PoolConfiguration record {|
     int maxActiveConnections = maxActiveConnections;
     int maxIdleConnections = maxIdleConnections;
     decimal waitTime = waitTime;
     int maxActiveStreamsPerConnection = maxActiveStreamsPerConnection;
+    int evictableIdleTime = 5 * 60;
```

Comment:
Since we are configuring this in seconds in ballerina and Java configuration is in milliseconds. Better to set it as a `decimal` value.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-http/pull/1841/commits/1b88b37adec5c89f232aa6b229cbfaa07613f4e1